sympy/combinatorics/perm_groups.py

```diff
@@ -694,6 +694,44 @@ def basic_transversals(self):
             self.schreier_sims()
         return self._transversals
 
+    def composition_series(self):
+        """
+        Add docs
+
+        """
+        if not self.is_solvable:
+            raise NotImplementedError('Group should be solvable')
+        der = self.derived_series()
+        series = [self]
+
+        for i in range(len(der)-1):
+            comp = []
+            low = der[i+1]
+            gns = low.generators
+            while low != der[i]:
+                while True:
+                    elm = der[i].random()
+                    if not elm in low:
+                        break
+
+                elm = [elm]
+                for pelm in elm:
```

Comment:
If the elm is in the form of `Permutation(1,2)(2,3)`, how can I extract both the elements in such a way so that further I can compute their order??

Reply:
This permutation is the cyclic permutation (1 3 2). It is the product of (1 2) and (2 3) but it can be represented as a product in many other ways.